src/data/WorkshopSchedule.js

```diff
@@ -13,6 +13,7 @@ const workshopSchedule = [
 		events: [
 			{
 				name: 'ACM-W × ACM Hack | Hackathon 101 + Team Formation',
+				duration: '30 min',
```

Comment:
OPTIONAL

This works, though you can store the duration as an integer here, and then use [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) to render the number as text.

If you decide to go down this path, make sure to change the property key to `durationMin`, so that the unit of the number is clear.

```js
const formatter = new Intl.NumberFormat('en-US',  {
	style: 'unit',
	unit: 'minute'
});
formatter.format(10); // returns '10 min'
```

Reply:
Pin for a later time, not necessary right now